daily-challenges/2025-01/18/problem.md

```
@@ -0,0 +1,7 @@
+# Playwright challenge
```

Markdownlint (0.37.0)

Expected: 1; Actual: 0; Below
Headings should be surrounded by blank lines

(MD022, blanks-around-headings)